includes/elements/container.php

```diff
@@ -331,12 +331,15 @@ protected function register_container_layout_controls() {
 		$this->add_group_control(
 			Group_Control_Flex_Container::get_type(),
 			[
-				'name' => 'container',
+				'name' => 'container_flex',
```

Comment:
Why not `flex`?

Reply:
Since there is also a section for the children's flex controls, so I don't want it to get messed.

Reply:
It's OK, the children have their prefix.

Reply:
Done